temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -235,6 +236,29 @@ message GetRegionResponse {
     temporal.api.cloud.region.v1.Region region = 1;
 }
 
+message GetNexusEndpointsRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100. 
+    int32 page_size = 1;
+
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
+
+    // optional, treated as an AND if specified
+    string target_namespace_id = 3;
+
+    // optional, treated as an AND if specified
+    string target_task_queue = 4;
+
```

Comment:
We can live without this for now and filter in the UI, I'd consider removing until there's demand.